build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -95,6 +95,13 @@ jobs:
       TargetFolder: '$(Build.ArtifactStagingDirectory)\AppXContents'
       flattenFolders: false
 
+  - powershell: |
```

Comment:
nit:  I have been using this syntax for inline powershell tasks:

  - task: PowerShell@2
    displayName: "displayname here"
    inputs:
      targetType: inline
      script: >
              "your script goes here"

Reply:
+1

Reply:
Yeah I did think this format was a bit odd, especially with the displayName at the end, but that is what the other examples had. I like your format better so I will switch to it.

---
In reply to: [520758307](https://github.com/microsoft/ProjectReunion/pull/284#discussion_r520758307) [](ancestors = 520758307)